hack/simplecov.rego

```diff
@@ -0,0 +1,52 @@
+# Copied from https://github.com/open-policy-agent/contrib/tree/main/simplecov/simplecov.rego
+package simplecov
+
+import future.keywords
+
+from_opa := {"coverage": coverage}
+
+coverage[file] = obj if {
+	some file, report in input.files
+	obj := {"lines": to_lines(report)}
+}
+
+covered_map(report) = cm if {
+	covered := object.get(report, "covered", [])
+	cm := {line: 1 |
+		some item in covered
+		some line in numbers.range(item.start.row, item.end.row)
+	}
+}
+
+not_covered_map(report) = ncm if {
+	not_covered := object.get(report, "not_covered", [])
+	ncm := {line: 0 |
+		some item in not_covered
+		some line in numbers.range(item.start.row, item.end.row)
+	}
+}
+
+to_lines(report) = lines if {
```

Comment:
nit: the [style guide](https://github.com/StyraInc/rego-style-guide#avoid-prefixing-rules-and-functions-with-get_-or-list) makes me think that `lines` is preferred over `to_lines`.